WebRoot/skins/_base/base4/skin.css

```diff
@@ -756,6 +756,28 @@ svg.ImgColumnDownArrow{
     fill: @IconColumnDownArrow@;
 }
 
+.ZmComposeView svg.ImgAttachClip {
+    width: 29px;
```

Comment:
.ZmComposeView svg.ImgAttachClip  is for the size of Image and 
div.AttachmentImgContainer > div.ImgAttachClip is the div container for positioning of the image to keep its postion wrt to the text which is below.